internal/common/roles/judge.go

```diff
@@ -0,0 +1,19 @@
+package roles
+
+import (
+	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+)
+
+// Judge is an interface that is implemented by baseJudge but can also be
+// optionally implemented by individual islands.
+
+type EvaluationReturn struct {
+	Rules       []rules.RuleMatrix
+	Evaluations []bool
+}
+type Judge interface {
+	PayPresident() (int, error)
+	InspectHistory() (map[int]EvaluationReturn, error)
+	DeclareSpeakerPerformance() (int, bool, int, bool, error)
+	DeclarePresidentPerformance() (int, bool, int, bool, error)
```

Comment:
that's a lotta return types--use a struct? tuple return types are actually not first class citizens in go.